sdk/avalon_sdk/fabric/fabric_work_order_receipt.py

```diff
@@ -0,0 +1,331 @@
+                logging.info("Invalid work order id {}".format(work_order_id))
+                return -1
+            if not is_valid_hex_str(
+                    binascii.hexlify(updater_id).decode("utf8")):
+                logging.info("Invalid updater id {}".format(updater_id))
+                return -1
+            params = []
+            params.append(work_order_id)
+            params.append(updater_id)
+            params.append(update_index)
+            receipt_result = self.__fabric_wrapper.invoke_chaincode(
+                self.CHAIN_CODE,
+                'workOrderReceiptUpdateRetrieve',
+                params)
+            if receipt_result is not None:
+                return work_order_result
+            else:
+                return -1
+        else:
+            logging.error(
+                "Fabric wrapper instance is not initialized")
+            return -1
+
+    def work_order_receipt_lookup(self, worker_service_id,
+                                  worker_id, requester_id, receipt_status):
+        """
+        Function to lookup receipts
+        Inputs:
+        worker_service_id is a Worker Service id whose receipts will be
+        retrieved.
+        worker_id is the Worker Id whose receipts are requested.
+        requester_id is the id of the entity requesting receipts.
+        receipt_status defines the status of the receipts retrieved.
+        id is used for json rpc request
+        Outputs:
+        total_count is the total number of receipts matching the lookup
+        criteria. If this number is bigger than the size of the ids array,
+        the caller should use a lookup_tag to call
+        work_order_receipt_lookup_next() to retrieve the rest of the
+        receipt ids.
+        ids is an array of the Work Order receipt ids that match the input
+        parameters.
+        """
+        if (self.__fabric_wrapper is not None):
+            if not is_valid_hex_str(
+                    binascii.hexlify(worker_id).decode("utf8")):
+                logging.info("Invalid worker id {}".format(worker_id))
+                return -1
+            if not is_valid_hex_str(
+                    binascii.hexlify(worker_service_id).decode("utf8")):
+                logging.info("Invalid service id {}".format(worker_service_id))
+                return -1
+            if not is_valid_hex_str(
+                    binascii.hexlify(requester_id).decode("utf8")):
+                logging.info("Invalid requester id {}".format(requester_id))
+                return -1
+            params = []
+            params.append(worker_service_id)
+            params.append(worker_id)
+            params.append(requester_id)
+            params.append(receipt_status)
+            receipt_result = self.__fabric_wrapper.invoke_chaincode(
+                self.CHAIN_CODE,
+                'workOrderReceiptLookUp',
+                params)
+            if receipt_result is not None:
+                return work_order_result
```

Comment:
should it return receipt_result or work_order_result ?

Reply:
Changed.